README.md

```diff
@@ -0,0 +1,49 @@
+# Conduit Connector NATS PubSub
+
+### General
+
+The [NATS](https://nats.io/) PubSub connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and a destination NATS PubSub connector.
+
+### Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.45.2
+- [NATS](https://nats.io/download/) 2.8.4 and [nats.go](https://github.com/nats-io/nats.go) library v1.16.0
+
+### How to build it
+
+Run `make`.
+
+### Testing
+
+Run `make test` to run all the unit and integration tests, which require Docker and Docker Compose to be installed and running. The command will handle starting and stopping docker containers for you.
+
+## Source
+
+### Connection and authentication
+
+The NATS PubSub connector connects to a NATS server or a cluster with the required parameters `urls`, `subject` and `mode`. If your NATS server has a configured authentication you can pass an authentication details in the connection URL. For example, for a token authentication the url will look like: `nats://mytoken@127.0.0.1:4222`, and for a username/password authentication: `nats://username:password@127.0.0.1:4222`. But if your server is using [NKey](https://docs.nats.io/using-nats/developer/connecting/nkey) or [Credentials file](https://docs.nats.io/using-nats/developer/connecting/creds) for authentication you must configure them via seperate [configuration](#configuration) parameters.
+
+### Receiving messages
+
+The connector listening on a subject receives messages published on that subject. If the connector is not actively listening on the subject, the message is not received. The connector can use the [wildcard](https://docs.nats.io/nats-concepts/subjects#wildcards) tokens such as `*` and `>` to match a single token or to match the tail of a subject.
```

Comment:
Yeah, exactly

Reply:
IMHO, it would be helpful to state it in a similar way in the readme, using the Conduit terminology (connector stopped, started etc.) instead of "actively listening" etc. I understand it's technically correct, but for improved clarity it's probably better.

Reply:
Updated the readme